📖 amp-story-shopping Added product json schema documentation (#38042)
ampproject/amphtml: extensions/amp-story-shopping/amp-story-shopping.md

}
```

> ### Product JSON Validation

Reviewer: ```suggestion
### Product JSON Schema
```

wdyt about using "schema" in the title so terminology us visible in the table of contents?

> The Product's JSON configuration will be validated using ajv against a schema [defined here](https://github.com/ampproject/amphtml/blob/main/examples/amp-story/shopping/product.schema.json). If validation fails on one or more of the shopping tags, an error message will be displayed, and the tag and product details / listing associated with the product(s) that have errors will not be rendered.

Reviewer: ```suggestion
See the schema for product JSON validation in [product.schema.json](https://github.com/ampproject/amphtml/blob/main/examples/amp-story/shopping/product.schema.json). 

If validation fails on one or more of the shopping tags, an error message will be displayed, and the tag and product details / listing associated with the product(s) that have errors will not be rendered.
```

Suggestion to follow the writing pattern used in `Validation` section. 

Is it important to mention ajv? If that's an implementation detail we might be able to leave it out.
If we keep it, it would be good to link to it's docs.

Author: I'm in favor of keeping it. JSON schemas have different validation rules depending on [different drafts](https://json-schema.org/specification.html) ajv has its own rules of which draft it will use (I believe the latest), but it's good info to know because not all drafts are backwards-compatible with each other, (i.e. some have different keywords), so if they are using an older draft, this could clue them in to upgrade.
